guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -98,6 +100,11 @@ class Config
 {% endcollapsible %}
 ---
 
+2.2.2. Factories SHOULD be used for object instantiation instead of `new` keyword. An object SHOULD be replaceable for testing or extensibility purposes.
+Exception: DTOs. There is no behavior in DTOs, so there is no reason for its replaceability.
```

Comment:
I would suggest it would be a good idea to either have an explanation for the term DTO here or link through to one.